experimental/packages/otlp-exporter-base/package.json

```diff
@@ -61,14 +61,16 @@
     "access": "public"
   },
   "dependencies": {
-    "@opentelemetry/core": "1.17.0"
+    "@opentelemetry/core": "1.17.0",
+    "pako": "^2.1.0"
```

Comment:
if pako is only needed for web or also for node.js?
as far as I know node.js has gzip support built in.

Reply:
For web there's also [Compression Streams API](https://developer.mozilla.org/en-US/docs/Web/API/Compression_Streams_API) which could avoid the extra bundle size of pako ([if browser support matches expectations](https://caniuse.com/mdn-api_compressionstream))